web/client/utils/ogc/WMC/index.js

```diff
@@ -276,8 +276,44 @@ export const toMapConfig = (wmcString, generateLayersGroup = false) => {
             };
             const zoom = parseFloat(get(msTagExtractor(globalExtensions, 'zoom'), 'charContent'));
 
+            const catalogServices = msTagExtractor(globalExtensions, 'CatalogServices');
+            const selectedService = attrExtractor(catalogServices, 'selectedService');
+            const services = msTagsExtractor(catalogServices, 'Service')
+                .map(catalogService => pickAttributeValues(
+                    catalogService,
+                    'serviceName',
+                    'autoload',
+                    'title',
+                    'type',
+                    'url',
+                    'format',
+                    'metadataTemplate',
+                    'oldService',
+                    'showAdvancedSettings',
+                    'showTemplate',
+                    'hideThumbnail'
+                ))
+                .reduce((resultObj, pickedService) => ({
+                    ...resultObj,
+                    [pickedService.serviceName]: {
+                        autoload: parseBoolean(pickedService.autoload),
+                        title: pickedService.title,
+                        type: pickedService.type,
+                        url: pickedService.url,
+                        format: pickedService.format,
+                        metadataTemplate: pickedService.metadataTemplate,
+                        oldService: pickedService.oldService,
```

Comment:
oldService is going to be removed, it is unuseful and so we should avoid to save it.